
Some invalid assumptions that fail since glibc-2.3.4.

  - Rene Rebe <rene@exactcode.de>

--- ./include/utils.h.orig	2002-09-24 09:19:22.000000000 +0200
+++ ./include/utils.h	2005-01-06 19:48:45.122345888 +0100
@@ -4,6 +4,9 @@
 #include "avm_default.h"
 #include "formats.h"
 
+#undef __THROW
+#define __THROW
+
 /**
  *  Utility functions.
  *    plain C functions are here
